services/app-web/src/pages/QuestionResponse/UploadResponse/UploadResponseForm.tsx

```diff
@@ -188,8 +121,8 @@ export const UploadResponse = () => {
                             variant="outline"
                             data-testid="page-actions-left-secondary"
                             disabled={apiLoading}
-                            link_url={`/submissions/${id}/question-and-answers`}
                             parent_component_type="page body"
+                            link_url={`/submissions/${id}/question-and-answers`}
                             onClick={() =>
                                 navigate(
                                     `/submissions/${id}/question-and-answers`
```

Comment:
These navigate urls need to switch to the rate Q&A page when uploading a rate. If you take a look at `UploadQuestionsForm`, I did the same thing there.